packages/wallet-sdk/src/CoinbaseWalletProvider.ts

```diff
@@ -23,6 +24,7 @@ interface ConstructorOptions {
   appLogoUrl?: string | null;
   appChainIds: number[];
   smartWalletOnly: boolean;
+  injectedSigner?: Signer;
```

Comment:
injectedSigner sounds ok.
how about `designatedSigner` or `preselectedSigner`?

Reply:
i tried to avoid this, cause user may not choose this signer, cause even Extension inject this signer in, user still will go to SCW popup and choose one, and that could be a different signer

Reply:
you know what.
i think we are about to make the same mistake as the previous SDKs.
if extension needs customization, they should do it via creating a subclass or something, instead of passing some constructor params.

i think we should just remove this option, and maybe introduce it later on the extension side.

Reply:
oh wow, you're right

Reply:
removed